niworkflows/interfaces/bids.py

```diff
@@ -533,6 +530,22 @@ class ReadSidecarJSON(SimpleInterface):
     layout = None
     _always_run = True
 
+    def __init__(self, fields=None, undef_fields=False, **inputs):
+        super(ReadSidecarJSON, self).__init__(**inputs)
+        self._fields = fields or []
+        self._undef_fields = undef_fields
+
+        if fields:
+            add_traits(self.inputs, fields)
```

Comment:
Why are you adding fields to the inputs? You don't seem to test or advertise this functionality.

Reply:
Ups, you are right - this doesn't make any sense.